arcade/text.py

```diff
@@ -613,6 +613,42 @@ def position(self, point: Point):
         else:
             self._label.position = x, y, self._label.z
 
+    @property
+    def tracking(self) -> float | None:
+        """
+        Get the tracking amount for this text object, or rather,
+        the added space between each character.
+
+        Returns a pixel amount, or None if the tracking is inconsistent.
```

Comment:
You can use google style docstring for the return here to make it more readable in docs

```
Returns:
    Explain the return value
```